test/cmdlineTests/recovery_ast_constructor/output

```diff
@@ -139,6 +140,7 @@ JSON AST (compact format):
                 "id": 12,
                 "mutability": "mutable",
                 "name": "",
+                "nameLocation": "-1:-1:18446744073709551615",
```

Comment:
The last number does not seem correct...

Reply:
That's the one I was referring to in the call. It's `numeric_limits<size_t>::max()`.